dev/sparktestsupport/shellutils.py

```diff
@@ -57,7 +55,12 @@ def run_cmd(cmd, return_output=False):
         if return_output:
             return subprocess_check_output(cmd).decode(sys.getdefaultencoding())
         else:
-            return subprocess_check_call(cmd)
+            popen = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+            for stdout_line in popen.stdout:
+                print(stdout_line.decode(sys.getdefaultencoding()), end='')
```

Comment:
BTW, it's best to stick to `utf-8` (see https://github.com/apache/spark/commit/2ec3265ae76fc1e136e44c240c476ce572b679df for instance)

Reply:
i was JUST going to comment on that lol.  :)

i'll take care of this tomorrow/early next week.

Reply:
or...  right now since i'm on a roll!